Beginners/Lesson One/README.md

```diff
@@ -0,0 +1,127 @@
+
+What this means is whatever you type will be run by Xcode and the results will be shown on the right. When Xcode is compiling code (whenever you change text), the top bar of the window will change to say "Running" and will show a little spinner. 
+
+Let's start by saying what _isn't_ code. You see the the first line, how it is greyed out? Well that's a comment. Comments are annotations in the code that are not ran by the compiler. We'll be using comments occasionally to skip bits of code, or to remind us what something is.
+
+OK let's write some code, let's try typing some things. 
+
+```swift
+1 + 2
+
+1 / 2
+1.0 / 2.0
+
+"Hello, Artsy"
+```
+
+You can see the results in the results pane:
+
+![Results](img/results.png)
+
+In programming, we want to work with abstractions. So let's make our first variable. We're going to make a simple calculator with a variable. What I'd like to do is define a variable called `a` and `b` then add them together to make `c`. This is a trivial example, but it's about foundations.
+
+```swift
+var a = 2
+var b = 3
+
+var c = a + b
+```
+
+Swift defines variables using `var`. While there is no type specified, the variable `a` is a `Int` (basically a round number; 1, 2, 3). Swift is a **strongly-typed language**, even though most of the time you don't actually need to say what type of thing it is. The compiler figures out the type of variables. 
+
+In Swift, `:` specifies a type. So the following two lines are equivalent.
+
+```swift
+var myString = "Hello, Artsy"
+var myString: String = "Hello, Artsy"
+```
+
+In this case we're dealing with a collection of letters, that combined, we call a string. This is common terminology in most programming languages, think of it as a string of letters.
+
+```swift
+var myString: String = 1
+```
+
+Swift will complain and say that `'Int' is not convertible to 'String'`. This is Swift's compile-time type checking. It prevents bugs from happening in apps at runtime by giving you errors before-hand. Swift's primary goal is safety against crashes in apps. 
+
+When we're writing code, very often we want to print out some useful information for someone to read. We're going to use this to show that some of our code is running. So lets add a print statment saying what the value of `c` is:
+
+```swift
+print(c)
+```
+
+With that working, we can use that to start doing some logic. We want to start checking how big our `c` is, and just like in English, we'll use an `if` statement:
+
+```swift
+if c > 6 {
+    print("Pretty big c you got there.")
+}
+```
+
+Let's unpack what we're seeing here. `if [something]` - what's happening here is that the bit after the `if` is checking to see if `c` is greater than the number (`Int`) `6`. If it is, then it will run the code inside the braces. 
+
+It is common to use a left margin / indentation to indicate that something is inside a braced section. It's not enforced though.
```

Comment:
Could give an example? 

```swift
if c > 6 {
            print("Pretty big c you got there.")
}
```

```swift
if c > 6 { 
print("Pretty big c you got there.") 
}
```

```swift
if c > 6 { print("Pretty big c you got there.") }
```

Mention that all of these are equivalent.